src/clib/pioc_support.c

```diff
@@ -235,17 +337,19 @@ void pio_log(int severity, const char *fmt, ...)
     rem_len = MAX_LOG_MSG - strlen(msg);
     strncpy(ptr, "\n\0", (rem_len > 0) ? rem_len : 0);
 
-    /* Send message to stdout. */
-    fprintf(stdout, "%s", msg);
-
     /* Send message to log file. */
     if (LOG_FILE)
```

Comment:
These changes seem incorrect. There will always be a LOG_FILE, once logging is initialized. There will never be a case where this code is reached and LOG_FILE is not set.